docs/security.rst

```diff
@@ -602,6 +602,38 @@ exclude them from add and edit form. Using our example you will define our view
         edit_columns = ['name']
 
 
+Password complexity validation
+------------------------------
+
+This feature only makes sense when using AUTH database.
+By default you can enable password complexity validation by setting `FAB_PASSWORD_COMPLEXITY_ENABLED = True`.
+
+This default enforces:
+
+- At least 2 Uppercase letters
+- At least 3 Lowercase letters
+- At least 1 special character
+- At least 2 numeric digits
+- At least 10 total digits
```

Comment:
It should be 'At least 10 total characters'.

Reply:
fixed, thks!

Reply:
LGTM, it has all the options I was looking for.